src/elements/content-sidebar/SidebarContent.scss

```diff
@@ -35,12 +40,20 @@
 
     .bcs-scroll-content {
         // Styles can be overriden by each sidebar
-        width: 323px; // 17px less than the total width for scroll bar
+        width: $sidebarScrollContentWidth; // 17px less than the total width for scroll bar
```

Comment:
Similarly here, we could move the comment to the line with variable declaration.